pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -501,12 +501,90 @@ func (e *DockercfgController) createDockerPullSecret(serviceAccount *v1.ServiceA
 		return nil, false, err
 	}
 	dockercfgSecret.Data[v1.DockerConfigKey] = dockercfgContent
+	dockercfgSecret.SetOwnerReferences([]metav1.OwnerReference{
+		{
+			Kind:       "Secret",
+			APIVersion: "v1",
+			Name:       tokenSecret.Name,
+			UID:        tokenSecret.UID,
+		},
+	})
```

Comment:
@deads2k are you aware of any reason we wouldn't want to have ownerrefs between the dockercfg secret and the token secret?  I'm sure we didn't do it originally because ownerrefs didn't exist, but now that they do can we just lifecycle the dockercfg secret w/ the tokensecret?  (modulo migration concerns)